saleor/graphql/account/mutations/service_account.py

```diff
@@ -1,8 +1,8 @@
 import graphene
 
-from saleor.core.permissions import get_permissions
-
 from ....account import models
+from ....core.enums import AccountPermissions
```

Comment:
I am not sure of that, `AccountPermissions` belongs more to `permission` file than `enum`.

Reply:
after talk, I moved `WebhookEventType` to event_types.py file
and restore permissions to previous state